maestro-android/src/androidTest/java/dev/mobile/maestro/MaestroDriverService.kt

```diff
@@ -287,6 +287,14 @@ class Service(
         responseObserver.onCompleted()
     }
 
+    override fun fetchSession(
+        request: MaestroAndroid.SessionRequest,
+        responseObserver: StreamObserver<MaestroAndroid.SessionResponse>
+    ) {
+        responseObserver.onNext(sessionResponse { id = SdkService.sessionInfo.sessionId })
+        responseObserver.onCompleted()
+    }
```

Comment:
I see changes here but not updated maestro server and maestro-android APKs

Reply:
How can I do that?

Reply:
Done